HISTORY.rst

```diff
@@ -3,6 +3,15 @@
 Release History
 ===============
 
+0.10.15
```

Comment:
Changes should go under `0.10.14` which is unreleased.

Reply:
fixed